.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
```

Comment:
🤔 We should add a pre-commit autoupdate to the CI pipeline.

Reply:
💯 added to DOCY (confluence) 👍